ws.py

```diff
@@ -280,6 +280,207 @@ def build_regions():
                 data.append(countries_from_geojson[region])
             else:
                 data.append(additional_regions[region])
+
+    countries_of_regions = {
+        "EU": [
+            "AUT",
+            "BEL",
+            "BGR",
+            "CYP",
+            "CZE",
+            "DEU",
+            "DNK",
+            "ESP",
+            "EST",
+            "FIN",
+            "FRA",
+            "GRC",
+            "HRV",
+            "HUN",
+            "IRL",
+            "ITA",
+            "LTU",
+            "LUX",
+            "LVA",
+            "MLT",
+            "NLD",
+            "POL",
+            "PRT",
+            "ROU",
+            "SVK",
+            "SVN",
+            "SWE",
+        ],
+        "African Group": [
+            "AGO",
+            "BDI",
+            "BEN",
+            "BFA",
+            "BWA",
+            "CAF",
+            "CIV",
+            "CMR",
+            "COD",
+            "COG",
+            "COM",
+            "CPV",
+            "DJI",
+            "DZA",
+            "EGY",
+            "ERI",
+            "ETH",
+            "GAB",
+            "GHA",
+            "GIN",
+            "GMB",
+            "GNB",
+            "GNQ",
+            "KEN",
+            "LBR",
+            "LBY",
+            "LSO",
+            "MAR",
+            "MDG",
+            "MLI",
+            "MOZ",
+            "MRT",
+            "MUS",
+            "MWI",
+            "NAM",
+            "NER",
+            "NGA",
+            "RWA",
+            "SDN",
+            "SEN",
+            "SLE",
+            "SOM",
+            "SSD",
+            "STP",
+            "SWZ",
+            "SYC",
+            "TCD",
+            "TGO",
+            "TUN",
+            "TZA",
+            "UGA",
+            "ZAF",
+            "ZMB",
+            "ZWE",
+        ],
+        "Northern America": ["CAN", "USA"],
+        "Australasia": ["AUS", "NZL"],
+        "Umbrella": ["AUS", "CAN", "ISL", "ISR", "JPN", "KAZ", "NOR", "NZL", "UKR", "USA"],
+        "G7": ["CAN", "DEU", "FRA", "GBR", "ITA", "JPN", "USA"],
+        "G20": ["ARG", "AUS", "BRA", "CAN", "CHN", "DEU", "FRA", "GBR", "IND", "IDN", "ITA", "JPN", "KOR", "MEX", "RUS", "SAU", "TUR", "USA", "ZAF"],
+        "FSM": ["MHL", "PLW", "NRU"],
```

Comment:
I'm not sure if this list is correct. Doesn't seem to match https://www.iso.org/obp/ui/#iso:code:3166:FM